cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -57,51 +54,15 @@ trait CsvImportExport {
   def ingestCSV(dataset: String, version: Int, csvPath: String, delimiter: Char) {
```

Comment:
IMO it's really nice for readers of code when functions have return types specified, especially when Unit